python/guide/performance/compression/float16.md

```diff
@@ -0,0 +1,243 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Mixed precision training using float16
+
+In this tutorial we will walk through how one can train deep learning neural networks with mixed precision on supported hardware. We will first see how to use float16 (both with Gluon and Symbolic APIs) and then some techniques on achieving good performance and accuracy.
+
+## Background
+
+The computational resources required for training deep neural networks have been increasing lately, because of growing of the complexity and size of models. Mixed precision training allows us to reduce the utilization of the resources by using lower precision arithmetic. In this approach you can train using 16 bit floating points (half precision) while using 32 bit floating points (single precision) for output buffers of float16 computation. It allows to achieve the same accuracy as training with single precision, while decreasing the required memory and training or inference time. Training using that combination of single and half precision is called mixed precision training.
```

Comment:
have been lately increasing because of growing complexity and model size.

by using lower precision arithmetic which is computationally less expensive and less costly in terms of space utilization.

Reply:
- you can train using 16 bit floating point
- while using 32 bit floating point (no s)
- This allows one to achieve the same accuracy ....

Reply:
"Training using that combination of single and half precision is called mixed precision training." redundant and can be removed.